esmvaltool/interface_scripts/interface.ncl

```diff
@@ -104,11 +106,13 @@ begin
     l_flag = True
     do jj = 0, natts - 1  ; loop over requested attributes
       if (.not.isatt(info[ii], atts(jj))) then
-        error_msg("f", scriptname, funcname, \
+        log_debug(scriptname + ", " + funcname + " - warning: " + \
```

Comment:
```suggestion
        error_msg("w", scriptname, funcname, \
```

Reply:
Those changes would be fine with me. I initially picked "log_debug" rather than "error_msg" because I think that these kind of warnings are rather extra information for the advanced user than real warnings and are therefore not needed be highly visible to all users.

Reply:
This is only for consistency with other interface functions.
Actually, we could probably always write warnings as `log_debug` in NCL, by changing these lines in `logging.ncl`:
https://github.com/ESMValGroup/ESMValTool/blob/26816d6274b50815b62782abea8eaca70a9e21f5/esmvaltool/interface_scripts/logging.ncl#L199-L202
Would that be a better solution?

Reply:
That could be an option even though that means that we do not have the possibility to generate (non-fatal) warning messages in NCL scripts anymore if I understand correctly.

Reply:
`error_msg("w", ...)` will still generate non-fatal messages, but using `log_debug` instead of `log_info`. The advantage of using the `error_msg` function is that it forces the user to specify the script and function names where it occurs, generating a more traceable message.

Reply:
Then let's change to log_debug for warning messages from NCL scripts and use error_mgs("w", ...) in select_metadata_by_atts!